src/store/combineStore.js

```diff
@@ -0,0 +1,8 @@
+import { createStore, combineReducers, applyMiddleware } from 'redux';
+import thunk from 'redux-thunk';
+
+export default () => {
+  const store = createStore(combineReducers({}), applyMiddleware(thunk));
```

Comment:
create a function to return a combined reducer because you may be combining more than ten reducers